gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -60,20 +65,20 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm,' '')"
+	backends+="$(usex libinput 'libinput,' '')"
+	backends+="$(usex x11-backend 'x11,' '')"
+	# Get rid of a trailing comma
+	backends="${backends%,}"
```

Comment:
Hell yeah! Finally managed to do it without sed!
The docs btw: https://devmanual.gentoo.org/tools-reference/bash/index.html#bash-string-replacements

Reply:
heh, ya, knew about bash string replacements, but it's just another version of sed to me :P

Reply:
I mean I read some QA article about sed and friends not being recommended to be used in a global scope (what's that?), so I decided that a "native" bash solution would be better. The principle is the same, but as you said, there are no other obvious ways to do it.

Reply:
It's faster to use native bash substitution and it's generally a bit cleaner as well. Thanks!

Reply:
Thanks to @thesamesam's [idea](https://github.com/gentoo/gentoo/pull/31029#discussion_r1198633580) managed to implement it more or less "nicely":
https://github.com/gentoo/gentoo/blob/ad83b87f3f988a2cf276e4a2659c241c48b0bfb1/gui-libs/wlroots/wlroots-0.16.2-r1.ebuild#L60-L66